### ISSUE TITLE
DNS highly unreliable inside guest VM

### ISSUE THREAD

**Issue by tomjn** [Parallels/vagrant-parallels#467]
Hello! I'm the maintainer of VVV, and I have issues provisioning using Parallels + arm boxes, specifically DNS issues.

Note that these issues _only_ occur when using the Parallels provider, the same provisioning, as well as the same boxes built for VirtualBox, and the same provisioners running in Docker all work without this issue.

These issues go away if I SSH in and configure the guest VM to ignore Parallels DNS and use Googles DNS instead. I can find no documentation on configuring DNS to avoid or solve this problem, nor have I found a way to automate replacing the DNS in the guest VM. Both boxes I've used are Ubuntu 20 Arm64 boxes, the latest being the Chef Bento box.

I've also had these issues for several years, across many versions of Parallels, and have raised support issues with the main Parallels team and sent technical information, but no progress or meaningful response has been made beyond the initial boilerplate requests.

My own tests indicate that the DNS server configured cannot reliably be pinged or connected to from within the VM

**Comment by cjlapao** [comment]
@tomjn We need a bit more information on this.

First, what do you mean Parallels DNS? We do not have a DNS per say and we rely on the bridge network to provide one.

I am assuming you change the DNS servers to 8.8.8.8 when you say you are using the Google DNS server

Could you please send us:
- What the output of `ipconfig` command in your host mac
- What the output of `ipconfig` command in your vms with issues
- Do you have a VPN/Router

This would help us understand the issue and see where we can fix it

**Comment by tomjn** [comment]
Also possibly useful, digging through ifconfig in the past I found the IP the VM thinks is the DNS server from the bridge, and found it unresponsive to pings/connections

**Comment by cjlapao** [comment]
Yes, I meant ifconfig in mac and in linux vms, the ipconfig for windows.

Ok, could you also let me know what is the version of Parallels Desktop you are running? we had a few issues with some VPNs when resolving bridges and that was sorted on the latest 19.3, while you do not have one, we did apply some fixes to the way ip's where assigned. So would be nice to know the version
